app/src/main/java/com/mamadou/newsapp/networking/NetworkStatusChecker.kt

```diff
@@ -6,13 +6,13 @@ import android.net.NetworkCapabilities
 
 class NetworkStatusChecker(private val connectivityManager: ConnectivityManager?) {
 
-    inline fun performIfConnectedToInternet(action: () -> Unit, notConnectedAction: () -> Unit) {
-        if (hasInternetConnection()) {
-            action()
-        } else {
-            notConnectedAction()
-            println("********* There is no Internet Connection **********")
-        }
+    inline fun performIfConnectedToInternet( action: () -> Unit, notConnectedAction: () -> Unit) {
+            if (hasInternetConnection()) {
+                action()
+            } else {
+                notConnectedAction()
+                println("********* There is no Internet Connection **********")
+            }
     }
```

Comment:
nit these spacing/indent changes can be reverted.

Reply:
Resolved1